crates/bevy_render/src/texture/fallback_image.rs

```diff
@@ -34,9 +45,10 @@ fn fallback_image_new(
     format: TextureFormat,
     dimension: TextureViewDimension,
     samples: u32,
+    value: u8,
 ) -> GpuImage {
-    // TODO make this configurable
-    let data = vec![255; format.pixel_size()];
+    // TODO make this configurable per channel
```

Comment:
Oh, that's my TODO, I completely forgot about that. Thanks for taking care of it. Or at least parts of it.